scenarios/node_scenarios_example.yml

```diff
@@ -3,11 +3,18 @@ node_scenarios:
     - node_stop_start_scenario
     - stop_start_kubelet_scenario
     - node_crash_scenario
+   #- stop_start_helper_node_scenario
     node_name:                                                      # node on which scenario has to be injected
     label_selector: node-role.kubernetes.io/worker                  # when node_name is not specified, a node with matching label_selector is selected for node chaos scenario injection
     instance_kill_count: 1                                          # number of times to inject each scenario under actions
     timeout: 120                                                    # duration to wait for completion of node scenario injection
     cloud_type: aws                                                 # cloud type on which Kubernetes/OpenShift runs
+    helper_node_ip:                                                 # ip address of the helper node
```

Comment:
Are we able to move these next couple of new parameters to the example part of docs/node_scenario.md? Might be less confusing since those items aren't going to be used in the actions that are run be default

Reply:
Yeah, moved them to the docs.